python/insights_ansible_playbook_signer/app.py

```diff
@@ -175,6 +175,9 @@ def sign_playbook(
         logger.debug(f"Play {i}/{len(raw_plays)} ('{play_name}'): OK.")
 
     logger.info("All plays were signed.")
+
+    # Handle None values in the output
+    yaml.add_representer(type(None), lib.represent_none)
```

Comment:
Since this changes the global behaviour of libyaml, I recommend moving it outside this function (which has a specific behaviour): this way it is not "hidden" in a particular function, and it is done at a clear point in the execution of the signer.

My suggestion would be to do this in `run()`, right after the argument parsing.